components/route/route-component.ts

```diff
@@ -357,6 +362,9 @@ export class BaseRouteComponent extends Vue {
 			const resolver = Resolver.startResolve(this.$options, to);
 			this.isRouteLoading = true;
 
+			// TODO: do we want to reset title suffix on route reload?
+			this.disableRouteTitleSuffix = false;
```

Comment:
do we?

Reply:
If reloading a route changes the route suffix for some reason (should only be done from within routeTitle) then yeah, we'd need this. Thing is, same as the comment above - I don't know if such a use case will ever be needed. It's not harmful to keep it in there either. May as well..?